nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -150,6 +169,8 @@ in
 
     environment.systemPackages = [ wrappedSlurm ];
 
+    services.munge.enable = true;
```

Comment:
Should it be possible to use `slurm` without `munge` ? If yes, please change the value to `mkDefault true` so users can easily override it in their config.

> However, you need to provide a key file to the munge module to make it work.

Can this change break existing configs that use `slurm` without  `munge` ? If yes, that should be documented in the release notes.

Reply:
You can run `slurm` without `munge` by turning off the authentication via `extraConfig`.
If you do not set `services.munge.enable=false` explicitly in that case, then this will result in the `munged.service` to fail on startup (given that no `service.munge.password` was provided).
I am not sure that this would really result in a breakage of an existing installation.
The way the `slurm` module is coded now `munge` is assumed to be there:
the `slurmctld.service` specifies `requires = [ "munged.service" ]`.
Munge is practially "hardcoded" into the nixos module already.

Reply:
Thanks for explaining. So there's no need for extra documentation.